components/Dialogs/src/MDCDialogPresentationController.m

```diff
@@ -17,29 +17,36 @@
 #import "MDCDialogPresentationController.h"
 
 #import "MaterialKeyboardWatcher.h"
+#import "MDCDialogTransition.h"
 #import "private/MDCDialogShadowedView.h"
+#import "private/MDCDialogTransitionMotionSpec.h"
+#import <MotionAnimator/MotionAnimator.h>
+#import <MotionTransitioning/MotionTransitioning.h>
 
 static CGFloat MDCDialogMinimumWidth = 280.0f;
 // TODO: Spec indicates 40 side margins and 280 minimum width.
 // That is incompatible with a 320 wide device.
 // Side margins set to 20 until we have a resolution
 static UIEdgeInsets MDCDialogEdgeInsets = {24, 20, 24, 20};
 
-@interface MDCDialogPresentationController ()
+@interface MDCDialogPresentationController () <MDMTransitionWithCustomDuration>
 
 // View matching the container's bounds that dims the entire screen and catchs taps to dismiss.
 @property(nonatomic) UIView *dimmingView;
 
 // Tracking view that adds a shadow under the presented view. This view's frame should always match
 // the presented view's.
```

Comment:
Nit: The comment on this property implies there's a separate "tracking view" that is not the "presented view". Could you please update the comment?

Reply:
Done.